pySDC/projects/Second_orderSDC/harmonic_oscillator_run_stability.py

```diff
@@ -0,0 +1,22 @@
+from pySDC.projects.Second_orderSDC.harmonic_oscillator_params import harmonic_oscillator_params
+from pySDC.projects.Second_orderSDC.stability_simulation import StabilityImplementation
+
+
+if __name__ == '__main__':
+    """
+    Damped harmonic oscillator as a test problem for the stability plot:
+        x' = v
+        v' = -kappa * x - mu * v
+        kappa: spring constant
+        mu: friction
+        Source: https://beltoforion.de/en/harmonic_oscillator/
+    """
```

Comment:
Why does this function tell us about the harmonic oscillator? I am not wondering what the harmonic oscillator does, but what this file is about. The documentation of the harmonic oscillator should go into the harmonic oscillator problem class and nowhere else.